package.json

```diff
@@ -31,6 +31,7 @@
 		"react": "18.2.0",
 		"react-dom": "18.2.0",
 		"react-icons": "^4.11.0",
+		"sharp": "^0.32.6",
```

Comment:
this is required for default image optimization loader.
https://nextjs.org/docs/pages/api-reference/next-config-js/output#automatically-copying-traced-files